woudc_data_registry/epicentre/instrument.py

```diff
@@ -67,18 +69,23 @@ def build_instrument(ecsv):
     serial = str(ecsv.extcsv['INSTRUMENT']['Number'])
     station = str(ecsv.extcsv['PLATFORM']['ID'])
     dataset = ecsv.extcsv['CONTENT']['Category']
+    contributor = str(ecsv.extcsv['CONTRIBUTOR']['ID'])
     location = [ecsv.extcsv['LOCATION'].get(f, None)
                 for f in ['Longitude', 'Latitude', 'Height']]
     timestamp_date = ecsv.extcsv['TIMESTAMP']['Date']
 
-    instrument_id = ':'.join([name, model, serial, station, dataset])
+    instrument_id = ':'.join([name, model, serial,
+                              station, dataset, contributor])
+    deployment = ':'.join([station, contributor])
+
     model = {
         'identifier': instrument_id,
         'name': name,
         'model': model,
         'serial': serial,
         'station_id': station,
         'dataset_id': dataset,
+        'deployment_id': deployment,
```

Comment:
The Instrument constructor expects a `contributor` key in its dictionary, not `deployment_id`. In this end this means we need to construct a `deployment_id` in less places, which is a good thing.